coralproject/talk
# [CORL-2775] match ban modal design

## What does this PR do?
This PR makes the Manage Ban Modal match the design spec.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. As a user with ban management permissions, open the ban modal either by clicking "Manage Ban" in a user row in the admin/community view, or by pressing "B" in the moderation queue
2. Observe that the ban modal more closely resembles the attached design picture
 
## How do we deploy this PR?
No special considerations necessary.
